### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by YuikoTakada**
This is work-in-progress for reviews by Japanese developers.
/assign @t-inu 
/assign @atoato88

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
 	"--container-runtime must be set to \"containerd\" or \"cri-o\" for rootless": "rootless のために、--container-runtime に「containerd」または「cri-o」を設定しなければなりません。",
 	"--kvm-numa-count range is 1-8": "--kvm-numa-count の範囲は 1～8 です",
 	"--network flag is only valid with the docker/podman and KVM drivers, it will be ignored": "--network フラグは、docker/podman および KVM ドライバーでのみ有効であるため、無視されます",
-	"--network flag is only valid with the docker/podman, KVM and Qemu drivers, it will be ignored": "",
-	"--network with QEMU must be 'user' or 'socket_vmnet'": "",
+	"--network flag is only valid with the docker/podman, KVM and Qemu drivers, it will be ignored": "--network フラグは、docker/podman, KVM および Qemu ドライバーでのみ有効であるため、無視されます",
+	"--network with QEMU must be 'user' or 'socket_vmnet'": "QEMU を用いる場合、 --network は、'user' か 'socket_vmnet' でなければなりません",
```
Comment (t-inu) [inline translations/ja.json:26]:
```suggestion
	"--network with QEMU must be 'user' or 'socket_vmnet'": "QEMU を用いる場合、--network は、'user' か 'socket_vmnet' でなければなりません",
```
nit: In this case, need not to add a space.

Comment (YuikoTakada) [reply translations/ja.json:26]: Thank you for your comments. I'll fix it

Comment (YuikoTakada) [reply translations/ja.json]: Done.

**Comment by atoato88** [comment]
/lgtm

I think it's OK to delete the "WIP" label.

**Comment by t-inu** [comment]
/assign @prezha @spowelljr 
Reviews by Japanese have finished.
Could you please merge this PR?

**Review by spowelljr — APPROVED** [review]

**Pull request merged**